From: Bryan Donlan <bdonlan@gmail.com>
Date: Wed, 11 May 2011 23:51:49 -0400
Subject: [PATCH] Fix autoconf error due to bad libtool invocation in configure.in

With autoconf >= 2.68, using A[CM]_PROG_LIBTOOL after AC_LIBTOOL_*
macros is illegal; violating this generates both a warning at aclocal
time and a failure at make time.
---
 configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 8dbf02d..22fe59f 100644
--- a/configure.in
+++ b/configure.in
@@ -64,10 +64,10 @@ AC_SUBST(LIBOFX_SO_AGE)
 
 AC_PROG_INSTALL
 
+AC_PROG_LIBTOOL
 AC_LIBTOOL_DLOPEN
 AC_LIBTOOL_WIN32_DLL
 AC_LIBTOOL_RC
-AM_PROG_LIBTOOL
 
 AC_ISC_POSIX
 AC_C_BIGENDIAN
-- 
